### DISCUSSION TITLE
爱发电疑似违规

### DISCUSSION THREAD

**Discussion by hejiehao** [Meloong-Git/PCL#4268] [讨论]
爱发电域名 afdian.net 被 clientHold 了，官网悬挂的 ICP 备案号经查询已注销，公安备案仍存在，网站类别为非交互式

官网域名备案被撤销，CDN 域名还在

**Comment by burningtnt** [comment]
补充：对应 IP 背后的服务器还在（154.8.151.42），但需要手动指定 DNS 解析

**Comment by SuoHouDaiShi** [comment]
新疆电信还可以正常访问，广西电信电脑版正常，手机版提示 404，目前不能确定消息真假
补充：
![8F8 RD2ZA G0_Z2GA{$NQ6E](https://github.com/user-attachments/assets/49b436c4-6e67-4421-9a20-f5667bfb6895)
再补充：关掉 [魔法] 之后再访问就会发现请求超时，即使再次开启也会无法访问

**Comment by burningtnt** [comment]
更新：154.8.151.42 已无法 ping 通

**Comment by SuoHouDaiShi** [comment]
更新：回光返照了？大多数地区又能正常访问了
补充：2024 07 16 / 00:37 复活仅仅不到一分钟，全国超时（运营商 DNS）
![图片](https://github.com/user-attachments/assets/2f8811b2-6bc9-4bdd-b4b7-0c5c670096c1)

**Comment by WorldHim** [comment]
似乎恢复了？

**Reply by allMagicNB** [reply]
no

**Comment by mcwuguodadi** [comment]
服务器迁移到国外了，国内的服务器被收回了：D
ifdian刚刚还能用，现在又403了

**Comment by lbyxiaolizi** [comment]
<img width="650" alt="Snipaste_2024-07-16_12-11-20" src="https://github.com/user-attachments/assets/8240c6b7-658c-425f-97a3-77dee949b797">
被公安干了

**Reply by LTCatt** [reply]
请不要在无依据的情况下揣测 “应该是洗钱方面的问题”

**Comment by lactobionicAcid** [comment]
爱发电目前有另一个备用网址`afdian.com`，或许PCL2可以将自带的赞助地址临时换为这个...?

**Comment by mcwuguodadi** [comment]
爱发电目前有2个临时网站，ifdian.net和afdian.com，但是备案应该几周内就会下发（顺利的话![1000062739.jpg](https://github.com/user-attachments/assets/c8010fe1-0495-4935-ab7f-fbbd5f3856fa)
